ni_measurementlink_service/measurement/info.py

```diff
@@ -45,12 +45,14 @@ class ServiceInfo(NamedTuple):
         As default we added a (str) description, (str) collection and a (List[str]) list of tags.
```

Comment:
>        As default we added a (str) description, (str) collection and a (List[str]) list of tags. Feel free to add your own Annotations as needed.

I'm not sure what this refers to and it reads a bit awkwardly. Where do we add these by default? I don't these add much clarity.

Reply:
Sphinx can read type hints, so types in docstrings are unnecessary. However, I didn't remove the types from the docstrings I made minor updates to.

Reply:
https://github.com/ni/measurementlink-python/pull/378